pages/dkp/kommander/2.1/operations/access-control/index.md

```diff
@@ -22,15 +22,15 @@ Use these two groups of resources to manage access control within 3 levels of sc
 | Workspace | Create namespaced Roles on the management cluster in the workspace namespace. | Federates ClusterRoles on all target clusters in the workspace.                            |
 | Project   | Create namespaced Roles on the management cluster in the project namespace.   | Federates namespaced Roles on all target clusters in the project in the project namespace. |
 
-The policies for each level and type create RoleBindings or ClusterRoleBindings on the clusters that apply to each category.
+The role bindings for each level and type create `RoleBindings` or `ClusterRoleBindings` on the clusters that apply to each category.
```

Comment:
Is "role bindings" really the right term change here.   The way I read this text, the user is creating "Kommander Roles" and "Cluster Roles" at the different 'levels', and then Kommander creates the RoleBindings/ClusterRoleBindings?     So the user is not creating 'role bindings'???

Reply:
Well, they are role bindings as we define them.

We changed the terms/phrasing in the UI specifically from policies to Role Bindings. So, whenever it was referencing Policies, we should have it changed to role bindings.

Reply:
Sorry, should have made it clearer.  I understand you were just updating 'policies->role bindings'.   I am wondering if the word 'policies' here was being used in a more generic sense before or perhaps it was misused or just wrong...

When I read this whole section in total, I don't see any reference in the words above it to either 'policies' _or_ 'role bindings'.   The first paragraph in this section says "Kommander has two conceptual groups of resources" and then mentions Kommander Roles and Cluster Roles, and then there is a table that describes how Kommander Roles are mapped to ClusterRoles based on what context they are in.    

I guess I'm saying that this sentence makes no sense to me with either subject!  What am I missing?

Reply:
ahh - the big reveal - I see what you're saying. I think John made a similar message below. I'll try to rework. Thanks for clarifying.

Reply:
I think that role bindings is more of a generic Kubernetes term - but this is an intro topic anyway, so it should be explained in some way.